docs/docs/tutorials/Generating stubs/using_protoc.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: default
+title: Generating stubs using protoc
+parent: Generating stubs
+grand_parent: Tutorials
+nav_order: 2
+---
+
+## Generating stubs using protoc
+
+1. Define your [gRPC](https://grpc.io/docs/) service using protocol buffers
+
+   `your_service.proto`:
+
+   ```protobuf
+    syntax = "proto3";
+    package your.service.v1;
+    option go_package = "github.com/yourorg/yourprotos/gen/go/your/service/v1";
+    message StringMessage {
+      string value = 1;
+    }
+
+    service YourService {
+      rpc Echo(StringMessage) returns (StringMessage) {}
+    }
+   ```
```

Comment:
I don't think this belongs on this page anymore. The section is called `Generating stubs`, I think we can safely assume that the user has a proto file ready at this point. Lets remove this step. We need to talk a little bit about the folder structure required to use `protoc` too. Like we're talking about calling `buf generate` from the root of the repository, `protoc` should be called from the root of the protobuf file hierarchy (the `proto` folder), to make it easy to resolve imports.